scripts/install.js

```diff
@@ -131,13 +114,23 @@ async function downloadLibtensorflow(callback) {
   await ensureDir(depsPath);
 
   const request = https.get(targetUri, response => {
+    var len = parseInt(response.headers['content-length'], 10);
+    var bar = new ProgressBar('downloading [:bar] :rate/bps :percent :etas', {
+      complete: '=',
+      incomplete: ' ',
+      width: 30,
+      total: len
```

Comment:
nit: I would just inline the `parseInt(response.headers...` part here.

Reply:
Done.